src/Symfony/Component/Console/Tests/Fixtures/DescriptorApplication2.php

```diff
@@ -20,5 +20,6 @@ public function __construct()
         parent::__construct('My Symfony application', 'v1.0');
         $this->add(new DescriptorCommand1());
         $this->add(new DescriptorCommand2());
+        $this->add(new DescriptorCommand3());
```

Comment:
For the record, i chose this approach on testing to include the case in other formats as well (Markdown, XML, etc.). This to avoid future regressions as well.